apps/api/src/app/notifications/dtos/activities-request.dto.ts

```diff
@@ -50,7 +51,24 @@ export class ActivitiesRequestDto {
     description: 'Page number for pagination',
   })
   @IsOptional()
-  page?: number;
+  @Type(() => Number)
+  @IsInt()
+  @Min(0)
+  page: number = 0;
+
+  @ApiPropertyOptional({
+    type: Number,
+    default: 10,
+    minimum: 1,
+    maximum: 50,
+    description: 'Limit for pagination',
+  })
+  @IsOptional()
+  @Type(() => Number)
+  @IsInt()
+  @Min(1)
+  @Max(50)
+  limit: number = 10;
```

Comment:
exposed the `limit` query param, before it was used internally with hardcoded 10 value